### PR TITLE
Add live scoring to the arena

### LINKED ISSUE #13: Add live scoring
Since we are already intending to have the scoring be automated by logging tokens moving through scoring zones in the token controller (#10), this issue is to incorporate that logic into the simulator so that the scores are displayed in real-time throughout the matches.

Note. If this is implemented in the same way as previously the scores will not be visible in the animations since display's textures are not tracked.

### PR DESCRIPTION
**Pull request by WillB97**
This uses 7-segment displays made from LEDs to allow the score displays to be visible in the animations as well. (CI will generate an animation and video)

Arena_37

fixes #13

### CODE REVIEW

**Review by Dasepho — APPROVED** [review]
That's awesome :smile:  :tada: 
Especially for the final this will be a great addition

File: controllers/token_scorer/token_scorer.py [hunk 1]
```diff
@@ -236,10 +283,43 @@ def process_token_locations(self) -> None:
...
+        return zone_scores
+
+    def update_displayed_scores(self) -> None:
+        scores = self.get_scores()
+
+        for zone, score in scores.items():
+            display = self.score_displays[zone]
+
+            score_low = score % 10
+            score_high = int(score / 10)
```
Comment (RealOrangeOne) [inline controllers/token_scorer/token_scorer.py]: it's not really low and high, it's more tens and units?

File: modules/shared_utils.py [hunk 2]
```diff
@@ -11,6 +13,13 @@ class Owner(IntEnum):
     ZONE_0 = 0
     ZONE_1 = 1
 
+    @classmethod
+    def zones(cls) -> Set[Owner]:
```
Comment (RealOrangeOne) [inline modules/shared_utils.py]: What's the purpose of this? Would `set(Owner)` not work?

Comment (WillB97) [reply modules/shared_utils.py]: This is just a relic of the previous way of differentiating beacons, removed in e1c0e195ba84c3f54bed936f45e7b991144a2649.

File: controllers/token_scorer/token_scorer.py (hunk 1 above)
Comment (WillB97) [reply controllers/token_scorer/token_scorer.py]: e3c0da681beb9f5cc7bdc5f10a4a12a69d6a7b98

**Pull request merged**